packages/lti-server/package.json

```diff
@@ -16,12 +16,13 @@
   ],
   "dependencies": {
     "@atomicjolt/lti-client": "@atomicjolt/lti-client",
-    "i18next": "^23.2.8",
-    "i18next-browser-languagedetector": "^7.1.0",
-    "jose": "^5.9.6"
+    "i18next": "^24.2",
+    "i18next-browser-languagedetector": "^8.0",
```

Comment:
Should this dependency even be in this package? I sort of assume it's supposed to just run client-side